examples/google-cloud-speech/output/google-cloud-speech/lib/google/cloud/speech.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
I am very nervous about enabling frozen strings by default. I worry about unintended consequences of the library returning string objects that users are unable change.

Reply:
I'm not too worried about it personally. But maybe I'm missing something?

@dazuma?

Reply:
Hmm. That's a good point, that we could potentially return frozen strings to users (thus being a breaking change if those strings weren't frozen before.) That said, it is the direction that Ruby is going, that String literals will likely be frozen by default, and in the cases where we care about returning mutable strings, I think it's on us to generate the code to ensure those strings are mutable. I'd rather take the opportunity to get ahead of this one.

Reply:
Strings won't be frozen by default until Ruby 3.0. We don't know what other breaking changes will be introduced to deal with the change in behavior at that point. I would not enable frozen strings by default until then.

Reply:
I would be less nervous about a change like this if we could run tests using the `--enable-frozen-string-literal` CLI argument and have them pass. But I've never gotten that to work.